src/modules/poweraccent/PowerAccent/PowerAccent.csproj

```diff
@@ -0,0 +1,27 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<OutputType>WinExe</OutputType>
+		<TargetFramework>net6.0-windows</TargetFramework>
+		<RuntimeIdentifiers>win-x64;win-arm64</RuntimeIdentifiers>
+		<UseWPF>true</UseWPF>
+		<Nullable>disable</Nullable>
+		<StartupObject>PowerAccent.Program</StartupObject>
+		<OutputPath>$(SolutionDir)$(Platform)\$(Configuration)\modules\PowerAccent</OutputPath>
+		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
+		<AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
+		<Platforms>x64</Platforms>
```

Comment:
@snickler this line should be removed for ARM support, correct?  line 13

Reply:
@crutkas - Yes. That line will make it compile solely for x64